en: tidb opreator offline installation (#517)
pingcap/docs-tidb-operator: en/deploy-tidb-operator.md

    {{< copyable "shell-regular" >}}

    ```shell
    helm upgrade tidb-operator pingcap/tidb-operator -f  ${HOME}/tidb-operator/values-tidb-operator.yaml
    ```

#### TiDB Operator Offline Installation

You need to install TiDB Operator offline by the following steps if your server cannot access the network:

1. Download `tidb-operator` chart

    If there is no external network on the server, you cannot configure the Helm repo to install TiDB Operator components and other applications. At this time, you need to download the chart file needed for cluster installation on a machine with an external network, and then copy it to the server.

    Use the following command to download the `tidb-operator` chart file:

    {{< copyable "shell-regular" >}}

    ```shell
    wget http://charts.pingcap.org/tidb-operator-v1.1.0.tgz
    ```

    Copy the `tidb-operator-v1.1.0.tgz` file to the target server and extract it to the current directory:

    {{< copyable "shell-regular" >}}

    ```shell
    tar zxvf tidb-operator.v1.1.0.tgz
    ```

> 2. Download Docker Images Used by TiDB Operator

Reviewer: ```suggestion
2. Download the Docker images used by TiDB Operator
```

>     If the server does not have an external network, you need to download and upload all Docker images used by TiDB Operator on the machine with an external network to the server, and then use `docker load` to install the Docker image on the server.

Reviewer: ```suggestion
    If the server has no access to the Internet, you need to download all Docker images used by TiDB Operator on a machine with Internet access and upload them to the server, and then use `docker load` to install the Docker image on the server.
```

    The Docker images used by TiDB Operator are:

    {{< copyable "shell-regular" >}}

    ```shell
    pingcap/tidb-operator:v1.1.0
    pingcap/tidb-backup-manager:v1.1.0
    bitnami/kubectl:latest
    pingcap/advanced-statefulset:v0.3.3
    k8s.gcr.io/kube-scheduler:v1.16.9
    ```

>     Among them, `k8s.gcr.io/kube-scheduler:v1.16.9` should be consistent with the version of your Kubernetes cluster, you don’t need to download it separately.

Reviewer: ```suggestion
    Among them, `k8s.gcr.io/kube-scheduler:v1.16.9` should be consistent with the version of your Kubernetes cluster. You do not need to download it separately.
```
Please avoid using `don't` in technical documentation. Use `do not` instead.

>     Next download all these images with the following command:

Reviewer: ```suggestion
    Next, download all these images using the following command:
```
`with` can have multiple meanings. Use `using` to avoid ambiguity.

    {{< copyable "shell-regular" >}}

    ```shell
    docker pull pingcap/tidb-operator:v1.1.0
    docker pull pingcap/tidb-backup-manager:v1.1.0
    docker pull bitnami/kubectl:latest
    docker pull pingcap/advanced-statefulset:v0.3.3

    docker save -o tidb-operator-v1.1.0.tar pingcap/tidb-operator:v1.1.0
    docker save -o tidb-backup-manager-v1.1.0.tar pingcap/tidb-backup-manager:v1.1.0
    docker save -o bitnami-kubectl.tar bitnami/kubectl:latest
    docker save -o advanced-statefulset-v0.3.3.tar pingcap/advanced-statefulset:v0.3.3
    ```

    Next, upload these Docker images to the server, and execute `docker load` to install these Docker images on the server:

    {{< copyable "shell-regular" >}}

    ```shell
    docker load -i tidb-operator-v1.1.0.tar
    docker load -i tidb-backup-manager-v1.1.0.tar
    docker load -i bitnami-kubectl.tar
    docker load -i advanced-statefulset-v0.3.3.tar
    ```

3. Configure TiDB Operator

>     TiDB Operator embeds a `kube-scheduler` to implement a custom scheduler. Please modify the `./tidb-operator/values.yaml` file to configure the Docker image name and version of this built-in `kube-scheduler` component, For example, the image used by `kube-scheduler` in your Kubernetes cluster is `k8s.gcr.io/kube-scheduler:v1.16.9`, please set `./tidb-operator/values.yaml` like this:

Reviewer: ```suggestion
    TiDB Operator embeds a `kube-scheduler` to implement a custom scheduler. To configure the Docker image's name and version of this built-in `kube-scheduler` component, modify the `./tidb-operator/values.yaml` file. For example, if `kube-scheduler` in your Kubernetes cluster uses the image `k8s.gcr.io/kube-scheduler:v1.16.9`, set `./tidb-operator/values.yaml` as follows:
```

    ```shell
    ...
    scheduler:
      serviceAccount: tidb-scheduler
      logLevel: 2
      replicas: 1
      schedulerName: tidb-scheduler
      resources:
        limits:
          cpu: 250m
          memory: 150Mi
        requests:
          cpu: 80m
          memory: 50Mi
      kubeSchedulerImageName: k8s.gcr.io/kube-scheduler
      kubeSchedulerImageTag: v1.16.9
    ...
    ```

>     Other items such as `limits`, `requests` and `replicas`, please modify as needed.

Reviewer: ```suggestion
    You can modify other items such as `limits`, `requests`, and `replicas` as needed.
```

4. Install TiDB Operator

>     Install TiDB Operator via the following instruction:

Reviewer: ```suggestion
    Install TiDB Operator using the following command:
```

    {{< copyable "shell-regular" >}}

    ```shell
    helm install ./tidb-operator --name=tidb-operator --namespace=tidb-admin
    ```

5. Upgrade TiDB Operator

>     If you need to upgrade the TiDB Operator, please modify the `./tidb-operator/values.yaml` file, and then execute the following command to upgrade:

Reviewer: ```suggestion
    If you need to upgrade TiDB Operator, modify the `./tidb-operator/values.yaml` file, and then execute the following command to upgrade:
```
